consensus/merge/src/test/java/org/hyperledger/besu/consensus/merge/TransitionProtocolScheduleTest.java

```diff
@@ -159,52 +155,45 @@ public void getByBlockHeader_returnsTimestampScheduleIfPresent() {
   }
 
   @Test
-  public void getByBlockNumber_returnsTimestampScheduleIfPresent() {
```

Comment:
Why are the getByBlockNumber tests being removed and migrated to cover getByBlockHeaderForBws?

This PR isn't removing getByBlockNumber so I think the tests should remain, we may need to duplicate some for the new method.

Reply:
Added back getByBlockNumber tests which are almost duplicates of the getByBlockHeader tests since that method still exists and is being used.